### DISCUSSION TITLE
🖼 Fanart Competition #6

### DISCUSSION THREAD

**Discussion by BirbSunflower** [sunflower-land/sunflower-land#3299] [Art]
For this competition, we encourage a wide range of art styles and techniques, not limited to pixel art. Feel free to explore your creativity and express yourself in any art style you choose!

Before submitting your artwork, we highly recommend that you read our fanart guidelines, which can be found here:

https://docs.sunflower-land.com/contributing/competitions/fan-art-competition

## Theme
The theme for this month is: **Flower Season** 🌷

Show us your creativity by designing something related to the chosen theme! Whether it's a character that you love, a scene that captivates you, or a moment that leaves a lasting impression, we want to see what you can come up with. Let your imagination soar and share your unique perspective with us! ❤

## Requirements ✏️

- AI-generated art will not be accepted.
- The art should be original and not plagiarised.
- By submitting art, you acknowledge you are the original creator of the art and you give Thought Farm and the community the rights to use it freely in the game and marketing purposes indefinitely.
- You agree to the [terms and services](https://docs.sunflower-land.com/contributing/competitions/fan-art-competition#terms-and-conditions) of the game.

When submitting art, please include the following:

- Your Farm ID.
- Your Discord ID.
- A image of your art with good quality.
- A NPC that represents you. 
- A Google Drive link with proof of your creation. 
To verify the authenticity of your artwork, we kindly request that you provide proof of your creative process. This may include work-in-progress images, concept sketches, time-lapse videos, and, in the case of digital art, we also request the original files.

Please note that the inclusion of NPCs will not have any impact on the judging process of the competition. However, if you choose to provide an NPC, it will be displayed in-game alongside your art piece. If you do not include an NPC, a random one will be selected. Also, a Bumpkin version of your NPC would be appreciated, but not required. 

You can use this image to showcase your NPC:

![image](https://github.com/sunflower-land/sunflower-land/assets/123522693/81132ce6-4863-4be7-a3ca-b5f78434ee60)

## Rewards 🏆
The winner will receive 1000 SFL and a runner-up will receive 500 SFL, however not only will the winning artwork be showcased on the official Sunflower Land Game social media channels and in-game, but it will also be seen by our vibrant community of players from around the world.

We also want to emphasize that even if you don't win, your design may still have the chance to be selected in the future. Our team will carefully review all submissions and may reach out to designers whose work stands out for future collaboration opportunities. Sunflower Land is a community game and we love showcasing and including artwork from our players. Now is your chance to have your artwork immortalised in the #1 Community Web3 game.❤

## Deadline 📅
Artists have until **February 1st** to submit their entries. After that, our art team will carefully review and analyze each submission to select the winners.

So, what are you waiting for? Grab your brushes, pens, or digital tools and get creating! Show us what you're made of and join our community of talented artists in celebrating the vibrant world of Sunflower Land Game. We can't wait to see what you come up with!

Good luck to all participants! 🎉

**Comment by clemrider** [comment]
Hi, I remember that I had posted directly here under the thread for an art competition about bumpkin and landscape. I had won a pink shirt with a paintbrush. I was very happy with that reward !
So here I am again. Just for fun and pleasure to draw and participate again.
I downloaded the game code from github, and started it on local host to see the new feature, and how should be the flowers. 
You'll find here a bunch of flowers ideas, inspired by what bees like here where I'm living =)
Field flowers, lavender, lusern, roses, poppies ... Bees also like make honey from pine trees, oranges trees or chestnuts trees.

 - Farm ID : 21662
 - Discord ID : szackass#4249
 - Drive : https://drive.google.com/drive/folders/1TVg0gyXj8xwwvssmFGDitz_tAQaCoJCl?usp=sharing
![flower2](https://github.com/sunflower-land/sunflower-land/assets/62701449/255f576e-06ad-47cd-8309-a5b31ccadc01)
![flower3](https://github.com/sunflower-land/sunflower-land/assets/62701449/a24a4d34-3245-4d80-99aa-5b37b3065e4f)
![Flower1](https://github.com/sunflower-land/sunflower-land/assets/62701449/46e4c6f3-cb9e-403b-b2c9-ee58725f3baa)
![flower_bed_modal_lavander](https://github.com/sunflower-land/sunflower-land/assets/62701449/f88c9c1f-cb29-43d6-8307-1c5d12f10115)
![flower_bed_modal_multiple](https://github.com/sunflower-land/sunflower-land/assets/62701449/14b8fbc8-608d-4d95-b0ff-4a23aae744a9)
![flower_bed_modal_sunny](https://github.com/sunflower-land/sunflower-land/assets/62701449/5aa866e1-b999-4f70-921a-def42c8ac214)
![flower_bed_modal_poppies](https://github.com/sunflower-land/sunflower-land/assets/62701449/8d4e8799-7660-4f98-90fe-91cf7b9fcc98)
![lavander7x12](https://github.com/sunflower-land/sunflower-land/assets/62701449/c6f499eb-884d-4fb8-942f-b1bcdd2040e7)
![lusern7x12](https://github.com/sunflower-land/sunflower-land/assets/62701449/45a185b5-903b-430e-85ef-695629e553c0)
![poppies7x12](https://github.com/sunflower-land/sunflower-land/assets/62701449/21668c9b-4343-476a-8d09-6ac8fd96d62d)


![NPC](https://github.com/sunflower-land/sunflower-land/assets/62701449/22f90d48-b64b-476d-8258-c6181f7399bc)

**Reply by BirbSunflower** [reply]
please check your email to give us access to the G. Drive folder 🤗

**Comment by jojothe2nd** [comment]
**Farm ID : 138
Discord ID : mr_findlay**

I always thought Pete had a crush on Betty....hope see loves the flowers he put so much effort into planting and tending with care!


![IMG_2906](https://github.com/sunflower-land/sunflower-land/assets/16280287/e5764be3-4a20-4cc1-b3a1-68948a159e97)

https://drive.google.com/file/d/1_PH97RgdEQcm6N82RUhMaEFx2KWjBD8N/view?usp=drive_link

![MrFindlayAvatar](https://github.com/sunflower-land/sunflower-land/assets/16280287/be62c237-ff01-4d80-896a-9d4d44cf374d)
![MrFindlayPFP](https://github.com/sunflower-land/sunflower-land/assets/16280287/0034916e-a89f-4ca8-a2f9-fe0d1fc7403f)

**Reply by BirbSunflower** [reply]
please check your email to give us access to the G. Drive folder 🤗

**Comment by kylaAdventures** [comment]
Hello everybody
not too much to say 😅
farm id:154245;
Discord id:shinobigoutcoco#8582
Link to the Time-lapse: https://www.youtube.com/watch?v=l2zZ607wvuU
![IMG_0322](https://github.com/sunflower-land/sunflower-land/assets/133995914/a0391626-640b-4442-ac9f-22bc07ef7233)
![IMG_0341](https://github.com/sunflower-land/sunflower-land/assets/133995914/4057c38c-8c9f-4df5-bf75-310fdff2f155)

**Comment by NycksonV** [comment]
`Farm ID: 136998`
`Discord: PoroTampinha #Nyck4168`

## Fã Art
`Lake nymph`
![Ninfa SFL](https://github.com/sunflower-land/sunflower-land/assets/118012559/7f421045-793d-4436-ba1a-01d329496f3c)
I hope it has to do with the season I made the nymph based on a bumpkin that appeared a while ago in one of Bird's lives, I loved her appearance so I tried to reproduce it in the drawing

##

## Drive
https://drive.google.com/file/d/17r_K1uPQw5ymw0INhsmHtjC4Cb1uBD25/view?usp=sharing

##

## Poro little Bumpkin
'This is me  u,u'
![Porinho](https://github.com/sunflower-land/sunflower-land/assets/118012559/289c37e7-3368-423a-8862-bff3b1f4e836)

| Sprite |
| --- |
![Porinho](https://github.com/sunflower-land/sunflower-land/assets/118012559/f0ed821a-1115-47cd-9eb8-95d2aa91186f)

##

## Sprite  ![Poro0](https://github.com/sunflower-land/sunflower-land/assets/118012559/db455ad6-eeff-4df9-a6e4-81093bd3d1f3)

##

**Reply by poteitouweb3** [reply]
Wow!!

**Reply by Elexandro** [reply]
uau!!, ficou show demais.

**Reply by ofcskill** [reply]
Nossa!! Ficou lindo demais.

**Comment by Vergelsxtn** [comment]
### **A box full of colors**

"I always wonder how NPC reacts to new items implemented in-game so I made this one"
![vergelsxtn-comp6](https://github.com/sunflower-land/sunflower-land/assets/99605039/e3e3bd9b-2d02-4a09-88db-6074139ff6bb)


An NPC that represents me:
![plague-doctor](https://github.com/sunflower-land/sunflower-land/assets/99605039/01826f58-09c6-4027-bff3-6010a3fbaf94)

A Google Drive link with proof of my creation: [Click here](https://drive.google.com/drive/folders/1q825Y5_eJnKKfK5ZAbQaAJhhmG5J9osZ?usp=sharing)

My Farm ID: 63
My Discord ID: Vergelsxtn

**Comment by andand0** [comment]
### Too much fertilizer

This is the reason why there is no more Rapid Growth...
<hr>


![too-much-fertilizer v1 (1)](https://github.com/sunflower-land/sunflower-land/assets/142132586/170929e0-5b81-4731-b72c-17238f218e4f)



<hr>

**Google Drive link:**  [Too much fertilizer process](https://drive.google.com/drive/folders/1f3A9ERxxB5rhyj96Fq1GaOEd893FeglH?usp=sharing)

`Farm ID: 165810`
`Discord: Andando` 

<hr>

### NPC  that represents me:


| Gif| Sprite|
|--------|--------|
|![Andando-NPC](https://github.com/sunflower-land/sunflower-land/assets/142132586/1db61c7b-500d-4ea0-9a78-021a8a0c3f88) | ![Andando-NPC-spritesheet](https://github.com/sunflower-land/sunflower-land/assets/142132586/827df8d2-24e2-4fee-9d2c-be40f267d637)|
